### ISSUE TITLE
Any example provided and other projects crash in DEBUG on mac

### ISSUE THREAD

**Issue by piotrus04** [resolume/ffgl#98]
Hi all !

all examples and any projects also i have all crash in debug mode.

in xcode log, i can read the error is a GL_INT == 0

any ideas ?

best

**Comment by thejoelpatrol** [comment]
Same with SDK 2.2, Avenue 7.24.1, macOS 15.7.1, arm64. I can run examples and copies of examples using Release, but not Debug. Trying to run a Debug build crashes when selecting the filter. I'm just going to use Release builds for now.

**Comment by piotrus04** [comment]
Yes, unfortunately resolume team abandoned ffgl sdk to prioritize Wire business.
For some of us this is really a big frustration because Wire is a great toy for approximative effects, but far from being a true language with versatile functions...

Very very far from vvvv, max or touchdesigner capabilities with open connections to other languages.
400 euros for a very simple GL language with NO serious documentation, just attractive UI because it is node based?
No comment...

Ffgl a non host based frameworks, that finally is just resolume based. And at the same time doesn't leave any developper participate to at least debug by itself the remaining issues because their code is so closed.
They are just making business ... very disappointing...
Looks like it is time for moving on.
